static/js/panels.js

```diff
@@ -83,7 +83,12 @@ export function initialize() {
     if (page_params.insecure_desktop_app) {
         open($("[data-process='insecure-desktop-app']"));
     } else if (page_params.server_needs_upgrade) {
-        open($("[data-process='server-needs-upgrade']"));
+        const avoid_notification = localstorage.supported() && ls.get("avoidNotificationFor7Days");
+        const seven_days_from_now = Date.now() + 7 * 86400000;
```

Comment:
Isn't something like: 
```javascript
const seven_days_from_now = getTime(addDays(new Date(), 7));
```
is better than hardcoding the milliseconds ?

Reply:
Thanks, @aryanshridhar!